docs/DeveloperGuide.md

```diff
@@ -124,6 +124,26 @@ The converter module consists of one class which converts the format of date and
 
 #### 2.2.3 BRANDON storage module
 
+The command module consists of 3 different classes. 
+The PatientList and AppointmentList classes act as data structures to store the records of Patient and Appointment 
```

Comment:
Inconsistent font type for classes. For example, PatientList and AppointmentList classes are normal font here but not in 2.2.4. Consider deciding on either font to make DG consistent.